### DISCUSSION TITLE
AR Orientation settings

### DISCUSSION THREAD

**Discussion by commit-master** [google/model-viewer#4450] [Q&A]
Hi,

First of all thank you for the awesome library that model-viewer is. While working on AR projects that involve lamps that I want to place to the ceiling I found something a bit painful to handle.

In the 360 viewer the model needs to be in one orientation while in AR mode it should literally be upside down. I've tried listening to the `ar-status` event but it does not seem to trigger anymore (at least on iOS), to change the orientation of the model back and forth but no luck.

Having the capability to change it would be really nice, or if there already is a way of doing it, please let me know !

Thanks !

**Comment by elalish** [comment]
Yeah, iOS is a bit strange with their orientation. You could try adding a click listener to your AR button and change the orientation then, since that's when the USDZ gets generated, and after that you can't affect it anymore.
